docs/en/user/parameters.rst

```diff
@@ -9,9 +9,9 @@ What is the meaning of the parameters?
    "database_name", "symfony", "name of your database"
```

Comment:
the default database name is `symfony` (see https://github.com/wallabag/wallabag/blob/master/app/config/parameters.yml.dist). You're right, we can change this default value.